content/docs/tasks/security/role-based-access-control/index.md

```diff
@@ -322,6 +322,179 @@ the "black" and "red" ratings in the "Book Reviews" section.
 
 > There may be some delays due to caching and other propagation overhead.
 
+## RBAC policy permissive mode
+
+RBAC permissive mode is for testing a new RBAC policy works as expected before
+rolling out to production. This sections shows how RBAC permissive mode is used.
+
+### Permissive mode for global RBAC configuration
+
+Before you start, please make sure that you have finished [preparation task](#before-you-begin).
```

Comment:
Maybe add a cleanup step in each section, so users can easily reset to to the blank slate.

Reply:
added.